apps/studio/src/pages/home/owners/OwnersTable.tsx

```diff
@@ -110,7 +120,7 @@ export default function OwnersTable({
   }
 
   if (isSuccess && collaboratorsData?.length === 0 && !query) {
```

Comment:
Just wanted to confirm the default behavior that currently exists: if getting the collaborator data is not a success then a table with default copy appears. If you agree something else should render on network/data query failure I can create a ticket to investigate further.